build.sbt

```diff
@@ -0,0 +1,25 @@
+import sbt.Keys._
+import sbt._
+
+val appName = "http-verbs-test"
```

Comment:
it's technically a lib name not an app, but if you can't be bothered to change I don't mind